docs/platform/release-notes/tizen-4-0-m1.md

```diff
@@ -1,10 +1,11 @@
 # Tizen 4.0 Public M1
 
-Release Date: 31 May, 2017
+Release date: May 31, 2017
```

Comment:
May. (Add a period at the end.)

Reply:
Nov. 01, 2017 is November 01, 2017, but May is May. In other words, May 31, 2017.